### DISCUSSION TITLE
Changelog

### DISCUSSION THREAD

**Discussion by Arytas** [pokepetter/ursina#591] [Q&A]
Is there a changelog for the new releases?

ATM i have a problem with the new update (5.3.0 -> 6.0.0) where i miss the window ui head think where i can drag and drop / close the game window.

![image](https://github.com/pokepetter/ursina/assets/9268623/9d810f5f-b65b-4d22-9c90-71b25d2c5d72)

![image](https://github.com/pokepetter/ursina/assets/9268623/f63e0604-45a8-4865-a77b-ac5bf8fbebf1)

I tried out different configurations, but was not able to figure out which one is the correct one. 
Also i cant see the window in my task bar anymore (also not visible in the Alt+Tab overview) 

So w/o changelog i am not shure if this is a problem / feature in the engine or if i miss the change :)

**Comment by Arytas** [comment]
Tried it out on my Windows PC, there i dont have the same issues

The problem exists for me on Linux (Ubuntu 22.04)

**Comment by onesome88** [comment]
pretty sure the outline (window UI head and borders) can be enabled / disabled via settings the `borderless` parameter in the `Ursina` class.
if set to False, shows the borders, if set to True, disables them.
Tl;DR `app = Ursina(borderless=False)` should make them appear again
